qiskit/primitives/base/base_estimator.py

```diff
@@ -111,68 +105,24 @@ class BaseEstimator(BasePrimitive):
+        *,
         options: dict | None = None,
     ):
         """
         Creating an instance of an Estimator, or using one in a ``with`` context opens a session that
         holds resources until the instance is ``close()`` ed or the context is exited.
 
         Args:
-            circuits: Quantum circuits that represent quantum states.
-            observables: Observables.
-            parameters: Parameters of quantum circuits, specifying the order in which values
-                will be bound. Defaults to ``[circ.parameters for circ in circuits]``
-                The indexing is such that ``parameters[i, j]`` is the j-th formal parameter of
-                ``circuits[i]``.
             options: Default options.
 
         Raises:
             ValueError: For mismatch of circuits and parameters list.
         """
-        if circuits is not None or observables is not None or parameters is not None:
-            warn(
-                "The BaseEstimator `circuits`, `observables`, `parameters` kwarg are deprecated "
-                "as of Qiskit Terra 0.22.0 and will be removed no earlier than 3 months after "
-                "the release date. You can use the 'run' method to append objects.",
-                DeprecationWarning,
-                2,
-            )
-        if isinstance(circuits, QuantumCircuit):
-            circuits = (circuits,)
-        self._circuits = [] if circuits is None else list(circuits)
-
-        if isinstance(observables, SparsePauliOp):
-            observables = (observables,)
-        self._observables = [] if observables is None else list(observables)
-
-        # To guarantee that they exist as instance variable.
-        # With only dynamic set, the python will not know if the attribute exists or not.
-        self._circuit_ids: dict[tuple, int] = self._circuit_ids
-        self._observable_ids: dict[tuple, int] = self._observable_ids
-
-        if parameters is None:
-            self._parameters = [circ.parameters for circ in self._circuits]
-        else:
-            self._parameters = [ParameterView(par) for par in parameters]
-            if len(self._parameters) != len(self._circuits):
-                raise ValueError(
-                    f"Different number of parameters ({len(self._parameters)}) and "
-                    f"circuits ({len(self._circuits)})"
-                )
-            for i, (circ, params) in enumerate(zip(self._circuits, self._parameters)):
-                if circ.num_parameters != len(params):
-                    raise ValueError(
-                        f"Different numbers of parameters of {i}-th circuit: "
-                        f"expected {circ.num_parameters}, actual {len(params)}."
-                    )
+        self._circuits = []
+        self._observables = []
+        self._parameters = []
         super().__init__(options)
 
-    ################################################################################
```

Comment:
I don't think this removal is related to this deprecation. I'm not a fan of this type of comment to indicate sections, but we need to discuss the removal carefully.

Reply:
Yeah. This is my bad. This was mistakenly approved during the review at the time of introduction. 
Let's discuss here.